rest/src/server/bootstrapper.js

```diff
@@ -93,23 +94,59 @@ module.exports = {
 
 	/**
 	 * Creates a REST api server.
-	 * @param {array} crossDomainConfig Configuration related to access control, contains allowed host and HTTP methods.
+	 * @param {array} config Application configuration
```

Comment:
{object} instead of {array}?

Reply:
seems like all the other param description end with '.'